### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by mithunkalan** [cross-rs/cross#596] [General]
I've been struggling for a few days trying to compile a mqtt for mips-musl until i used your project. pretty straightforward and worked great. So thank you to the team!

**Comment by WNacci** [comment]
Wanted to second this- cross greatly simplified compilation to an ARMv6 computer for a small hobby project of mine, thanks to the team!

**Comment by Eduardo-RFarias** [comment]
Hours lost, mental health strained in the battle of cross-compilation until stumbling upon this gem of a crate. Now, it's simply a love story!
A heartfelt thank you to every single developer involved.
